codegen/aws-typescript-models/smithy-build.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "1.0",
+  "plugins": {
+    "typescript-codegen": {
+      "service": "com.amazon.rdsdataservice#RdsDataService",
+      "package": "@aws-sdk/client-rds-data-node",
```

Comment:
Can this be generated off of the `sdkId`?

Reply:
Yes. This file is hand written for now, with the plan of generating the file in its entirety once we've gotten Gradle iterating over the files stored in `model/`.